packages/expo-image/ios/EXImageBorders.m

```diff
@@ -0,0 +1,215 @@
+// Copyright 2020-present 650 Industries. All rights reserved.
+
+#import <expo-image/EXImageBorders.h>
+#import <React/RCTUtils.h>
+
+static const CGFloat EXImageViewBorderThreshold = 0.001;
+
+EXImageBorder EXImageBorderMake(CGFloat width, CGColorRef color, RCTBorderStyle style)
+{
+  EXImageBorder border;
+  border.width = width;
+  border.color = color;
+  border.style = style;
+  return border;
+}
+
+EXImageBorders EXImageBordersInit()
+{
+  EXImageBorders borders;
+  EXImageBorder border = EXImageBorderMake(-1, nil, RCTBorderStyleUnset);
+  borders.all = EXImageBorderMake(-1, nil, RCTBorderStyleSolid);
+  borders.top = border;
+  borders.right = border;
+  borders.bottom = border;
+  borders.left = border;
+  borders.start = border;
+  borders.end = border;
+  return borders;
+}
+
+void EXImageBordersRelease(EXImageBorders borders)
```

Comment:
Why use this functioney pattern? Wouldn't making this sth like a`EXImageViewBorderManager` class with instances of `EXImageViewBorder` a little bit clearer, easier to maintain, override, etc?

Reply:
Yes you're probably right here. This started our as a couple helper functions for structs but grew out to much more. Will refactor this 👍

Reply:
I've refactored the CornerRadii & Borders code into separate classes. More code has been removed from the View into these helper classes.